src/View/Home/home.html.twig

```diff
@@ -0,0 +1,57 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}accueil{% endblock %}
+    
+
+{% block content %} 
+
+<div class = "backgroundPage">
+  <div id="carouselExampleFade" class="BOOTSTRAP carousel carousel slide carousel-fade BOOTSTRAP carousel" data-bs-ride="carousel">
```

Comment:
Why do you this class "BOOTSTRAP" ?? It doesn't exists.
Please delete everywhere you use it